lib/redis_client.rb

```diff
@@ -686,7 +686,7 @@ def connect
   rescue ConnectionError => error
     raise CannotConnectError, error.message, error.backtrace
   rescue CommandError => error
-    if error.message.include?("ERR unknown command `HELLO`")
+    if error.message.include?("ERR unknown command 'HELLO'")
```

Comment:
Thanks but we need to actually test both. The backquote vs single quote is a Redis 7.0 change. Boths can happen...

Reply:
Yes but wouldn't a Redis 7 server always have the HELLO command available since it's been available Redis 6?

Reply:
In fact if backquotes in the error message was a Redis 7 change, we are using this test to raise an UnsupportedServer error telling people they should be on Redis 6+. I don't think we would want to swallow the CommandError and raise UnsupportedServer error if they were on Redis 7 cause that is a compatible Redis version right? So we should only test for the message how it's output in versions below Redis 6 since that's when the HELLO command was added to Redis.

Reply:
This is actually my issue right now, we have a redis instance on Heroku that's reporting redis version 6.2.6 but we are getting the `ERR unknown command 'HELLO'`.